Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -528,9 +599,14 @@ def http_request(
         if response.status_code in {202, 204}:
             # Delete channel or remove user from channel return 204 if successful
             # Update message returns 202 if the request has been accepted for processing
+            # Remove member from a channel returns 204 if successful.
+            # Create channel with a membershipType value of shared, returns 202 Accepted response code
+            # and a link to the teamsAsyncOperation.
             return {}
         if response.status_code == 201:
-            # For channel creation query, we get a body in the response, otherwise we should just return
+            # For channel creation query (with a membershipType value of standard or private) returns 201 if successful,
+            # Create chat returns 201 if successful
+            # we get a body in the response, otherwise we should just return
```

Comment:
No need for the comment about `response.content`.
```suggestion
            # For chat creation and channel creation query (with a membershipType value of standard or private) returns 201 if successful,
```